test/robust/compare/output1__Stub.js

```diff
@@ -35,12 +63,36 @@ describe('/', function() {
           return;
         }
 
-        body.should.have.property('name');
+        /*eslint-disable*/
+        validator.validate(body, schema).should.be.true;
```

Comment:
@mohsen1 eslint has been throwing on this line because its an 'unused statement'. other people have had trouble with eslint and chai.should/expect and say just to comment it out